dfxdemo/dfxdemo.py

```diff
@@ -1,3 +1,4 @@
+#updated 29 july 2022 to allow for filter choices args
```

Comment:
Can you remove this comment? This information would be present in the Git history